Prior to 2.7.4, the bsddb module was a no-op, as it did not get built.
Starting with 2.7.4, the behavior is different, and it causes tests to
dump core.  So skip it.  The module is removed in Python 3, so this patch
is not being submitted upstream.

Note, that changes in setup.py are applied inside patch 02-setup.patch
(# raise db_found).

--- Python-2.7.14/Makefile.pre.in.orig	2018-01-26 03:09:44.649652970 -0800
+++ Python-2.7.14/Makefile.pre.in	2018-01-26 03:10:11.112938557 -0800
@@ -1076,7 +1076,7 @@
 		ensurepip ensurepip/_bundled \
 		json json/tests \
 		sqlite3 sqlite3/test \
-		logging bsddb bsddb/test csv importlib wsgiref \
+		logging csv importlib wsgiref \
 		lib2to3 lib2to3/fixes lib2to3/pgen2 lib2to3/tests \
 		lib2to3/tests/data lib2to3/tests/data/fixers lib2to3/tests/data/fixers/myfixes \
 		ctypes ctypes/test ctypes/macholib \
